### ISSUE TITLE
States map is not working properly

### ISSUE THREAD

**Issue by cziter15** [royto/logbook-card#83]
**Describe the bug**
After update to 2.0.0 it seems that icon states map is not working properly.

**To Reproduce**
1. Use states_map and modify icon and icon color
2. Notice that states map not working

**Expected behavior**
Working as ealier, priror to 2.0.0

**Screenshots**
2.0.0
![image](https://github.com/royto/logbook-card/assets/5003708/8c17e7d9-5166-468d-8e07-302806bf3ca1)
1.13.0
![image](https://github.com/royto/logbook-card/assets/5003708/bea705e7-120c-4220-b773-20f293e12dee)

**Environment (please complete the following information):**
 - HA Version: 2023.11.2
 - Browser: Chrome
 - Card version: 2.0.0

**Card configuration**
```
entity: sensor.pelletmon_pcb_sensor_burner_status_name
title: ''
max_items: 5
type: custom:logbook-card
state_map:
  - icon: mdi:alert-remove
    value: unavailable
    icon_color: '#822'
  - icon: mdi:fire-off
    value: off
    icon_color: '#aaa'
show:
  separator: true
  end_date: true
  start_date: true
  icon: true
desc: true
no_event: '-'
custom_logs: false

```

**Comment by royto** [comment]
Thanks for the report

Fixed in [2.0.1](https://github.com/royto/logbook-card/releases/tag/2.0.1)

**Comment by cziter15** [comment]
Confirmed, working as expected. Thanks!
